src/counter.rs

```diff
@@ -131,11 +136,22 @@ impl Counter {
             return Err(ErrorKind::NoEventsFound.into());
         }
 
+        // Add all voters we haven't encountered so far.
+        let mining_keys: Vec<Address> = self.call_poa(get_validators_fn::call())?;
+        for mining_key in mining_keys {
+            let voter = self.call_key_mgr(get_voting_by_mining_fn::call(mining_key))?;
+            if voter.is_zero() {
+                vprintln!("Voting key for {} is zero. Skipping.", mining_key);
```

Comment:
What is `vprintln!`? I've never seen this, it's not in the [`std` macros](https://doc.rust-lang.org/std/index.html#macros)?

Reply:
It's just defined locally in this function: It prints if `verbose` is `true`.